discocss

```diff
@@ -34,8 +34,16 @@ EOF
 
 ln -f -s "$preloadFile" /tmp/discocss-preload.js
 
-sed -i 's|  // App preload script, used to provide a replacement native API now that|try { require\(`/tmp/discocss-preload.js`) } catch \(e\) {console.error\(e\);} |' \
+if [ "$(uname)" = "Darwin" ]; then
+    which gsed 1>/dev/null || echo 'run "brew install coreutils" to have GNU/sed for macOS'
+    gsed -i 's|  // App preload script, used to provide a replacement native API now that|try { require\(`/tmp/discocss-preload.js`) } catch \(e\) {console.error\(e\);} |' \
```

Comment:
```suggestion
    sed -i '' 's|  // App preload script, used to provide a replacement native API now that|try { require\(`/tmp/discocss-preload.js`) } catch \(e\) {console.error\(e\);} |' \
```

Would this work with MacOS' sed?

Reply:
It doesn't work. macOS sed/anytool has always been a pain to work with, so generally devs are suggested to use gsed/g<tool> available thorugh brew or just compiling yourself

Reply:
```suggestion
    LC_ALL=C sed -i '' 's|  // App preload script, used to provide a replacement native API now that|try { require\(`/tmp/discocss-preload.js`) } catch \(e\) {console.error\(e\);} |' \
```
This should work, I tried it on MacOS 11.2.1.

Reply:
@mlvzk that worked